### ISSUE TITLE
Pd emscripten as a .js package

### ISSUE THREAD

**Issue by DaviRaubach** [cuinjune/purr-data#2]
Hi there, and congratulations on your project!
I would like to make some web-based art running a pd patch and relating visual and sound parameters. 
How can I use just your libpd "emscripted" to run my patches?

**Comment by cuinjune** [comment]
@DaviRaubach Hi, I'm not sure if I understand what you mean by "use libpd emscripted to run patches". 

Please check the following projects and see if any suit your needs:

- https://mathr.co.uk/empd/#empd
- https://github.com/cuinjune/PdWebParty
- https://github.com/cuinjune/Ofelia

**Comment by Proeditor07** [comment]
hey is this official repo of purr data?

**Comment by cuinjune** [comment]
@Proeditor07 Nope, this repo only contains the web app build of Purr Data. (for running in the browser)
The following is the official Purr Data repository: https://git.purrdata.net/jwilkes/purr-data

**Comment by DaviRaubach** [comment]
I am a little bit late here. But anyway, thank you @cuinjune 
That is what I need.
